### PR TITLE
Adding Source Code to support Long Accessions

### PR DESCRIPTION
**Pull request by anbhimi**

### CODE REVIEW

File: modules/cold-extraction/ColdDataRetriever.py [hunk 1]
```diff
@@ -587,6 +595,8 @@ def run_cold_extraction():
     ap.add_argument("--ThirdIndex", default=config['ThirdIndex'], type=int,
                     help="Set the CSV column index of third index query attribute. "
                          "Required only if the number of query attributes is 3")
+    ap.add_argument("--LongAccession", default=config['ThirdIndex'], type=bool,
```
Comment (pradeeban) [inline modules/cold-extraction/ColdDataRetriever.py:598]: This should be, 
default=config['LongAccession']

File: modules/cold-extraction/config.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
 	"SecondIndex": 5,
 	"ThirdAttr": "StudyDate",
 	"ThirdIndex": 3,
+	"LongAccession": true,
```
Comment (pradeeban) [inline modules/cold-extraction/config.json:13]: default should be false.

File: modules/cold-extraction/ColdDataRetriever.py (hunk 1 above)
Comment (anbhimi) [reply modules/cold-extraction/ColdDataRetriever.py]: Modified the source code as per comments

File: modules/cold-extraction/config.json (hunk 2 above)
Comment (anbhimi) [reply modules/cold-extraction/config.json]: Modified the source code as per comments

**Pull request merged**